jujugui/static/gui/src/app/components/user-profile/test-user-profile.js

```diff
@@ -494,7 +501,20 @@ describe('UserProfile', () => {
     var instance = component.getMountedInstance();
 
     assert.equal(jem.listEnvironments.callCount, 1);
-    assert.deepEqual(instance.state.envList, models);
+    assert.deepEqual(instance.state.envList, jemModels);
+
+    // Make sure we properly displayed the different bits within a JEM model.
+    var output = component.getRenderOutput();
+    var displayedModel = output.props.children.props.children.props.children[1].props.children[0].props.children[1].props.children[1][0]; // eslint-disable-line
```

Comment:
gross - maybe we should investigate some of those libraries for querying react components for situations like this.